charts/scalardl/README.md

```diff
@@ -76,6 +76,14 @@ Current chart version is `5.0.0-SNAPSHOT`
 | ledger.strategy.type | string | `"RollingUpdate"` | New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate |
 | ledger.tls.caRootCertSecret | string | `""` | Name of the Secret containing the custom CA root certificate for TLS communication. |
 | ledger.tls.certChainSecret | string | `""` | Name of the Secret containing the certificate chain file used for TLS communication. |
+| ledger.tls.certManager.dnsNames | list | `["localhost"]` | Subject Alternative Name (SAN) of a certificate. |
+| ledger.tls.certManager.duration | string | `"87600h0m0s"` | Duration of a certificate. |
```

Comment:
It is set to a value greater than the default value, where does this value come from? If ledger supports hot reloading of certificates, then a shorter duration is better.

Reply:
Ah, sorry. I set the default values based on our test environment (because it's a test environment, the duration value is very long.)

However, I didn't confirm the ScalarDL support hot reloading of certificates. So, first, I will ask the development team whether ScalarDL supports hot reloading or not.

If ScalarDL supports hot reloading, I will set the default value to empty (i.e., we will use the default value of `Certificate` spec).

If ScalarDL does not support hot reloading, I will discuss what value is better as a default value again.

Reply:
@superbrothers  
FYI:

ScalarDL does not support hot reloading. So, we decided to set `8760h (365 days)` as a default duration.

However, we are planing to support hot reloading on the ScalarDL side. After we implement hot reloading, we will update this default value to `2160h (90 days)`.